Lib/test/test_capi/test_misc.py

```diff
@@ -2157,10 +2157,6 @@ def test_gilstate_matches_current(self):
 
 
 class Test_testcapi(unittest.TestCase):
-    locals().update((name, getattr(_testcapi, name))
-                    for name in dir(_testcapi)
-                    if name.startswith('test_') and not name.endswith('_code'))
```

Comment:
It should not be removed, only simplified. Remove the part after `and`.

Reply:
What's the purpose of keeping them, if there are no tests that they load?

Reply:
I'm sure there are still tests that they load.

Reply:
Oh, there are two cases. This one has tests:

```python
>>> import _testcapi
>>> len([(name, getattr(_testcapi, name))
...                     for name in dir(_testcapi)
...                     if name.startswith('test_') and not name.endswith('_code')])
126
```

But, `Test_testcapi` in `Lib/test/test_capi/test_getargs.py` can be deleted:

```python
>>> len([(name, getattr(_testcapi, name))
...                     for name in dir(_testcapi)
...                     if name.startswith('test_') and name.endswith('_code')])
0
```